critiquebrainz/frontend/templates/artist/entity.html

```diff
@@ -181,3 +194,24 @@
   </div>
 </div>
 {% endblock %}
+
+{% block scripts %}
+  {{ super() }}
+  <script>
+  $(".no_reviewed_entities").hide();
```

Comment:
With javascript it's a good idea to run commands only after the full page has loaded. You can use document.ready here: https://github.com/metabrainz/critiquebrainz/blob/483594644b4954c3a62f89c84d9abf1e97b14a8a/critiquebrainz/frontend/templates/review/modify/base.html#L63